### ISSUE TITLE
code for Image classification pytorch

### ISSUE THREAD

**Issue by bplm** [lucidrains/vit-pytorch#40]
Hi!! Awsome work!!
the code that you used (vit_pytorch for classifying catsanddogs) is for image classification....i referred to the code...it runs model on train and validation set...if We have a separate test set...how to run model on it and generate confusion matrix??Also,there is no provision of checkpt and callbacks...

Also, in my case i am classifying cancer as benign and malignant..and when I run code similar to the one u did on validation set...it shows all predicted labels and targets as the SAME..Pls help!!!!
